__main__.py

```diff
@@ -1,2 +1,25 @@
+import argparse
+
 if __name__ == "__main__":
-    print("oh, what up?")
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--api", help="Get data from API", type=bool,
```

Comment:
at this point, would adding any of these modifiers to our call actually do anything or are you just setting them up so we can have them do specified things in the future?